doc-content/enterprise-only/cases/case-management-closing-cases-ref.adoc

```diff
@@ -2,6 +2,8 @@
 = Closing cases
 
 A case instance can be completed when there are no more activities to be performed and the business goal is achieved, or it can be closed prematurely. Usually the case owner closes the case when all work is completed and the case goals have been met.
```

Comment:
Why are we telling people that you can close it prematurely? Is there a way to tell? What should they do if it is closed before it ought to be?

Reply:
Good question. Sent the same out for QE to clarify. I can ASSuME why, but I will check 1st.